src/ORM/Query.php

```diff
@@ -43,7 +43,7 @@
  * @method \Cake\ORM\Table getRepository() Returns the default table object that will be used by this query,
  *   that is, the table that will appear in the from clause.
  * @method \Cake\Collection\CollectionInterface each(callable $c) Passes each of the query results to the callable
- * @method \Cake\Collection\CollectionInterface sortBy($callback, int $dir) Sorts the query with the callback
+ * @method \Cake\Collection\CollectionInterface sortBy(callable|string $path, int $order = SORT_DESC, int $sort = \SORT_NUMERIC) Sorts the query with the callback
```

Comment:
```suggestion
 * @method \Cake\Collection\CollectionInterface sortBy(callable|string $path, int $order = SORT_DESC, int $sort = SORT_NUMERIC) Sorts the query with the callback
```
This should stop phpcs from whining.

Reply:
CS doesnt like it it seems:
```
Error: Class name \Cake\ORM\ in @method should be referenced via a fully qualified name.
Error: Class name \Cake\ORM\ in @method should be referenced via a fully qualified name.
```